Components/LandingContact.tsx

```diff
@@ -1,12 +1,35 @@
 import Image from "next/image";
+import emailjs from 'emailjs-com';
+import { useRef, useState } from "react";
 
 const LandingContact = () => {
+  const form = useRef();
+  const [isMessageSent, setIsMessageSent] = useState(false);
+
+  const sendEmail = (e) => {
+    e.preventDefault();
+
+    emailjs
+      .sendForm('service_rjqxpou', 'template_f8vouol', form.current, 'o2kgKJm8TdvQP4pBr')
```

Comment:
```suggestion
      .sendForm('service_rjqxpou', 'template_f8vouol', form.current, 'o2kgKJm8TdvQP4pBr')
```
Please document the things here using comments as these are hard-to-understand things.